go.mod

```diff
@@ -1,10 +1,14 @@
-module github.com/rbretecher/go-postman-collection
+module github.com/vbisbest/go-postman-collection
```

Comment:
Please revert all changes made to this file except the `go 1.15` :) 

You can run `go mod tidy` for that.